frontend/src/components/transactions/ReceiptRow.tsx

```diff
@@ -1,18 +1,24 @@
+import BN from "bn.js";
 import React from "react";
 import { Row, Col } from "react-bootstrap";
 
-import * as T from "../../libraries/explorer-wamp/transactions";
+import ActionRow from "./ActionRow";
 
+import * as T from "../../libraries/explorer-wamp/transactions";
+import Gas from "../utils/Gas";
+import Balance from "../utils/Balance";
+import { truncateAccountId } from "../../libraries/formatting";
 import { displayArgs } from "./ActionMessage";
```

Comment:
Moved the first line to the last line and merged it with the group according to the CONTRIBUTING.md:

```suggestion
import * as T from "../../libraries/explorer-wamp/transactions";
import Gas from "../utils/Gas";
import Balance from "../utils/Balance";
import { truncateAccountId } from "../../libraries/formatting";
import { displayArgs } from "./ActionMessage";
import ActionRow from "./ActionRow";
```